src/aggregation/bucket/term_agg.rs

```diff
@@ -379,9 +392,16 @@ impl SegmentAggregationCollector for SegmentTermCollector {
 
         let mem_pre = self.get_memory_consumption();
 
-        bucket_agg_accessor
-            .column_block_accessor
-            .fetch_block(docs, &bucket_agg_accessor.accessor);
+        if let Some(missing) = bucket_agg_accessor.missing_accessor1 {
```

Comment:
I don''t understand what happens when we have two columns of type string and int
For instance, if we have 3 docs.

doc1 -> "hello"
doc2 -> 3
doc3 -> missing

And we want a term aggregation with missing = "N/A".
With logic below, don't we end up with 

```
"hello" -> 1
3 -> 1
"N/A" -> 3 (instead of 1)
```

Reply:
I added a comment for this on the aggregation request